content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -8,19 +8,35 @@ type: docs
 description: >
   You can find a list of available commands below.
 ---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --c` or `mastercontroller --commands` displays all the available options and a brief guide about their usage.
```

Comment:
I think what you meant is `mastercontroller --help`. This is what the commands option is used for: https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#non-interactive-mode-scripted-commands

Reply:
--help is working the same as --c or --commands. Both display the options and their usage if used in that way. But I'll update it as --help.